tests/drivers/dma/chan_link_transfer/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(chan_blen_transfer)
```

Comment:
should this be `chan_link_transfer`?

Reply:
yes, just copy out of thinking from chan_blen_transfer, updated